modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -769,58 +770,79 @@ const Quirks = (editor: Editor): Quirks => {
     return (!sel || !sel.rangeCount || sel.rangeCount === 0);
   };
 
-  // All browsers
-  removeBlockQuoteOnBackSpace();
-  emptyEditorWhenDeleting();
+  if (Rtc.isRtc(editor)) {
```

Comment:
Instead of a massive if else block, would this be better/more readable if we created 2 functions (eg `setupRtc` and `setup`)? We could then do something like this:
```
if (Rtc.isRtc(editor)) {
  setupRtc();
} else {
  setup();
}

Reply:
Yeah, that is probably easier to read. And would make it easier to yank out later.